examples/preview-panel-in-modal/style.css

```diff
@@ -0,0 +1,148 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: rgb(244, 244, 249);
+  color: rgb(65, 65, 65);
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  padding: 1rem;
+}
+
+.container {
+  margin: 0 auto;
+  max-width: 640px;
+  width: 100%;
+}
+
+.aa-Detached {
+  --aa-spacing-quarter: calc(var(--aa-spacing-half) / 2);
```

Comment:
I'm not sure we want to extend the Autocomplete CSS var namespace in our demos. This could easily conflict or get us confused.

Reply:
I'm fine with inlining. Fixed in 750f1d88.